src/Glpi/Helpdesk/DefaultDataManager.php

```diff
@@ -169,6 +169,17 @@ private function addFormTranslations(int $forms_id, string $name, string $descri
             $translated_description = $TRANSLATE->translate($description, 'glpi', $lang);
             if ($translated_name !== $name) {
                 $form_stmt->execute([Form::TRANSLATION_KEY_NAME, $lang, json_encode(['one' => $translated_name]), $name_hash]);
+            } else {
+                $DB->insert(
+                    ItemTranslation::getTable(),
+                    [
+                        'itemtype'     => Form::class,
+                        'items_id'     => $forms_id,
+                        'language'     => $lang,
+                        'key'          => Form::TRANSLATION_KEY_NAME,
+                        'translations' => '{}',
+                    ]
+                );
```

Comment:
1. The statement should be used here.
2. I guess the name hash must be properly defined.

```suggestion
                $form_stmt->execute([Form::TRANSLATION_KEY_NAME, $lang, '{}',  $name_hash]);
```